tests/components/application-connector/test/application-gateway/runner_test.go

```diff
@@ -0,0 +1,37 @@
+package application_gateway
+
+import (
+	"context"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var applications = []string{"positive-authorisation", "path-related-error-handling", "kubernetes-resources-error-handling"}
+
+func (gs *GatewaySuite) TestCases() {
+
+	for _, app := range applications {
```

Comment:
Should we maybe do `gs.Run` for each `app`, considering that app grouping is meaningful?